extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -107,14 +110,25 @@ export class AmpSidebar extends AMP.BaseElement {
     screenReaderCloseButton.classList.add('-amp-screen-reader');
     // This is for screen-readers only, should not get a tab stop.
     screenReaderCloseButton.tabIndex = -1;
-    screenReaderCloseButton.addEventListener('click', () => {
-      this.close_();
-    });
+    screenReaderCloseButton.addEventListener('click', () => this.close_());
     this.element.appendChild(screenReaderCloseButton);
 
     this.registerAction('toggle', this.toggle_.bind(this));
     this.registerAction('open', this.open_.bind(this));
     this.registerAction('close', this.close_.bind(this));
+
+    this.element.addEventListener('click', e => {
+      const target = closestByTag(dev().assertElement(e.target), 'A');
+      if (!target) {
```

Comment:
Check that `target` has a `href`, too. Also, this code's [duplicated](https://github.com/ampproject/amphtml/blob/553492cf2c54e9c974ddea12b3c524a03b68de0b/src/document-click.js#L116-L124).